clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/model/KubernetesAutoscalerStatus.groovy

```diff
@@ -33,7 +33,6 @@ class KubernetesAutoscalerStatus {
     if (autoscaler.status == null) {
       log.warn("Autoscaler on ${autoscaler.metadata.name} has a null status. The replicaset may be missing a CPU request.")
     } else {
-      this.currentCpuUtilization = autoscaler.status.currentCPUUtilizationPercentage
```

Comment:
Should we remove `currentCpuUtilization` from the model (line 25) since we are no longer setting it? I just did a quick grep through the code base and it doesn't look like that property is ever used.

Reply:
Oops -- that's exactly what I meant to do. Let me fix that now.

Reply:
Ah sorry although that property isn't used in Clouddriver, it looks like Deck does expect that property to be set for the UI: https://github.com/spinnaker/deck/blob/master/app/scripts/modules/kubernetes/src/serverGroup/details/details.html#L151

I'm wondering if the `currentCpuUtilization` can be read from the `currentMetrics` of the HPA status instead? https://static.javadoc.io/io.fabric8/kubernetes-model/4.1.1/io/fabric8/kubernetes/api/model/HorizontalPodAutoscalerStatus.html#getCurrentMetrics--

Reply:
Oh -- good catch! 

I see this under currentMetrics: `this.currentCpuUtilization = autoscaler.status.currentMetrics.resource.currentAverageUtilization`

Fabric8 is sort of a pain to dig through. Let me verify that this is the correct property.

Reply:
@mrampton that looks promising! Really, really appreciate you verifying this.

Reply:
Hi @maggieneterval -- believe that should do the trick. 

Sorry for the delay, went down a rabbit hole looking at deck...

Reply:
@mrampton yaaay, thank you so much! If there was a #spinfluencer of the day award you would win. I'll cherry-pick this into 1.15 so it can go out with the next patch release, most likely Monday.